llvm/test/Transforms/EntryExitInstrumenter/pre-inliner-instrumentation.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -passes="default<O1>" -S < %s | FileCheck %s
```

Comment:
should also test `O0` versions of these, plus `lto-pre-link`

Reply:
Done, I think, but it is evening, and I might be missing something.